libs/wingc/src/type_check.rs

```diff
@@ -57,15 +61,24 @@ pub enum SymbolKind {
 	Namespace(Namespace),
 }
 
+// Information about a variable in the environment
 #[derive(Debug, Clone)]
 pub struct VariableInfo {
+	// Type of the variable
 	pub _type: TypeRef,
+	// Can the variable be reassigned?
 	pub reassignable: bool,
+	// The phase in which this variable exists
+	pub flight: Phase,
```

Comment:
What would it mean for a variable to exist in an "independent" phase / how do I create such a variable? Maybe add some explanation

Reply:
We currently don't support independent phase so I'm not sure. The general idea is that if you have a:
```wing
struct Foo {
  a: num
}
```
Then there's nothing preventing us from using the type `Foo` in either phase (and therefore also exchanging it between them. In that case `a` can be defined phase independent.
I don't feel like adding any documentation about this yet since there are too many unclear aspects and we don't support it yet.